app/templates/main/userProfile.html

```diff
@@ -158,10 +158,10 @@ <h3 class="accordion-header" id="headingThree">
                    <td>
                        {% if row.completedTraining or not trainingList %}
                            <i class="bi bi-check-lg"></i> &nbsp&nbsp
-                           <a class="trainingPopover" id="{{row.program.id}}" href="#" data-toggle="popover" title="Program Trainings" data-content= "{{trainingListView()}}">View</a>
+                           <a class="trainingPopover" id="{{row.program.id}}" href="#!" data-toggle="popover" title="Program Trainings" data-content= "{{trainingListView()}}">View</a>
```

Comment:
Rather than adding something to the url we want to change the link behavior. Currently if a users first action on a page is to click the view button it breaks other interactions/notifications.